docs/developer-guide/CLI Methods/generating-the-bundled-config-file.md

```diff
@@ -0,0 +1,87 @@
+# Generating the Bundled Configuration File
+
+[Running Basemaps Locally][running-basemaps-locally] \> [Configuring the Basemaps/server package][configuring-the-basemapsserver-package] \> [Providing a path to a bundled configuration JSON file][path-to-json-config-file] \> Generating the Configuration File
+
+This guide shows you how to generate the bundled configuration file using the **basemaps/cli** package.
+
+## Prerequisites
+
+### AWS Credentials
+
+You'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled configuration file required for running the **Basemaps/server** package.
+
+## Cloning the Repository
+
+### basemaps-config
+
+You must clone the **basemaps-config** repository to your machine.
```

Comment:
```suggestion
Clone the **linz/basemaps-config** repository to your machine.
```

must is a strong word here as I believe you can use remote access to the config, eg storing it in S3 s3://some-config/basemaps-config/